tests/unit/agent_status_test.py

```diff
@@ -355,11 +418,26 @@ def test_basic(self):
 (these statistics cover the period from Fri Sep  5 11:14:13 2014 UTC)
 
 Bytes uploaded successfully:               10000
-Last successful communication with Scalyr: Fri Sep  5 23:13:13 2014 UTC
-Last attempt:                              Fri Sep  5 23:13:13 2014 UTC
-Last copy request size:                    10000
+Last requests:                             All successful
 Health check:                              Good
 
+Api keys statistics
```

Comment:
Also, what happens in the common case where you only have one api key?  Do we still print these headers?  I would actually special case it so that it looks like how it does now.